### ISSUE TITLE
corvus-dotnet.yml should not auto-merge SpecFlow.* in Corvus.Testing

### ISSUE THREAD

**Issue by idg10** [endjin/.github#36]
The `corvus-dotnet.yml` file in `/repos/live` specifies non-standard auto-merge wildcards:

https://github.com/endjin/.github/blob/e5be9503ce83730c46620101a49e67e9ca899de2/repos/live/corvus-dotnet.yml#L49-L52

It's the `SpecFlow.*` part that is non-standard.

This doesn't work, because SpecFlow tends to release sets of packages simultaneously, all of which need to be taken at once. This is why we needed the metapackage in the first place.

The effect of this non-standard wildcard setup is that the first of the dependabot PRs for each SpecFlow update to build successfully gets auto-approved, but there's typically another one that fails. We then end up needing to create a manual PR with all the updates in one batch. And then, when that gets merged, the PR that pr-autoflow auto-approved hangs around, instead of being dismissed as out of date automatically like it should, because dependabot can see that dependjinbot has meddled with it, and decides it shouldn't delete a PR that someone else has edited.

So in summary, this doesn't automate the work we need, and it creates additional clean-up work. We'd be better off with `Corvus.Testing` just having the same wildcards as everything else.

**Comment by JamesDawson** [comment]
Closed by #40
